nnsvg.c

```diff
@@ -35,7 +32,7 @@ static int nnsvg_new(lua_State *L) {
     input = luaL_checklstring(L, 1, &len);
 
     // Second arg specified what the string is: by default, a filepath
-    bool is_svg_data = false;
+    int is_svg_data = 0;
```

Comment:
I'd really rather we not perpetuate a history of using terrible data types, so, C99 bool, please ;p.

Reply:
(We require C99/C++11, so, not a compat issue).